packages/core/src/components/panel-stack/_panel-stack.scss

```diff
@@ -13,7 +13,6 @@
   display: flex;
   flex-shrink: 0;
   align-items: center;
-  z-index: 1;
```

Comment:
Why was this removed? This breaks the box-shadow from the next line when the panel view's content has a background.

Reply:
@invliD It was quite a while ago, but it seems it got moved to `.#{$ns}-panel-stack-view` below. If it's a problem at the moment, I think you should file a new issue about that.

Reply:
I did see it got added to the view, but that doesn't explain why it was removed here. I added it here for exactly this reason when I wrote the panel stack; guess I'll just add it back…